app/views/system_mailer/config_test.html.erb

```diff
@@ -3,8 +3,8 @@
 </h2>
 
 <p>
-	Check the sender and from email addresses to make sure they're right.a
+	Check the sender and from email addresses to make sure they're right.
 </p>
 <p>
-	This link should go to the collection list: <%= link_to 'click me', { controller: 'admin', action: 'uploads', only_path: false } %>
+	This link should go to the <%= link_to 'collection list', { controller: 'admin', action: 'collection_list', only_path: false } %>.
```

Comment:
LGTM

We would prefer using rails path helper rather than the `{ controller: ..., action: ... }` syntax here. In fact we are refactoring instances of that in other areas of the code.

Reply:
Thanks for reviewing! I used the URL helper, since we need the full URL in the email.